frontend/sass/pages/_movies.scss

```diff
@@ -0,0 +1,251 @@
+          font-size: 1.6rem;
+        }
+      }
+
+      .basic-data {
+        display: flex;
+        flex-direction: column;
+        gap: 20px;
+        height: 100%;
+
+        @media only screen and (max-width: abstracts.$large-width) {
+          .data-row {
+            flex-wrap: wrap;
+            gap: 10px !important;
+          }
+        }
+
+        .data-row {
+          display: flex;
+          flex-direction: row;
+          gap: 20px;
+
+          p {
+            font-size: 1rem;
+            letter-spacing: normal;
+          }
+
+          .yellow {
+            color: map.get(map.get(abstracts.$highlight-colors, "yellow"), "default");
+          }
+
+          .cyan {
+            color: map.get(map.get(abstracts.$highlight-colors, "cyan"), "default");
+          }
+        }
+
+        .movie-description {
+          max-width: 1000px;
+        }
+      }
+
+      .all-buttons {
+        display: inline-flex;
+        align-items: center;
+        justify-content: space-between;
+        margin-top: 40px;
+
+        svg {
+          fill: abstracts.$primary-light;
+          cursor: pointer;
+        }
+
+        .buttons,
+        .transparent-buttons {
+          display: inline-flex;
+          align-items: center;
+          flex-wrap: wrap;
+          gap: 10px;
+
+          & > * {
+            max-width: max-content;
+          }
+
+          button {
+            display: inline-flex;
+            align-items: center;
+            max-height: fit-content;
+            min-width: fit-content;
+            gap: 5px;
+          }
+        }
+
+        .transparent-buttons {
+          justify-content: flex-end;
+
+          button {
+            color: abstracts.$primary-light;
+            padding: 0;
+          }
+
+          .yellow {
+            color: map.get(map.get(abstracts.$highlight-colors, "yellow"), "default");
+
+            svg {
+              fill: map.get(map.get(abstracts.$highlight-colors, "yellow"), "default");
+            }
+          }
+
+          .pink {
+            color: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
+
+            svg {
+              fill: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
+            }
+          }
+        }
+      }
+    }
+
+    @media only screen and (max-width: abstracts.$large-width) {
+      .all-buttons {
+        display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+        gap: 10px;
+      }
+    }
+  }
+}
+
+.movie-streaming-sites {
+  display: inline-flex;
+  align-items: center;
+  justify-content: flex-end;
+  background-color: abstracts.$primary-darkgrey;
+  width: 100%;
+  min-height: 10px;
+  padding: 10px;
+  gap: 10px;
+}
+
+.rotating-star {
+  svg {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 50px;
+    height: 50px;
+    margin: -25px 0 0 -25px;
+    -webkit-animation: spin 4s linear infinite;
+    -moz-animation: spin 4s linear infinite;
```

Comment:
Näitä -moz tai -webkit juttuja ei varmaan enään tarvitse https://caniuse.com/?search=animation